processes/SRCSRV_REQUEST_CREATE.BOSS_handle_SR.pdef

```diff
@@ -34,6 +34,13 @@ Ruote.process_definition :name => 'BOSS_handle_SR', :revision => '0.2' do
     # Any error will get notified by this flanked suprocess
     do_log_error :flank => true
 
+    obsticket :action => 'require', :lock_project => 'SR_${ev.id}'
+    _if :test => '${__result__} != true' do
+      sequence do
+        do_log :msg => 'Failed to get request lock, bailing out'
```

Comment:
s/request/project/ as it's a generic lock on the project.

Also "bailing out" sounds bad, Maybe "Failed to get project lock. Assuming a multi-request promotion is being processed and letting the other process handle it."

Reply:
Changed the log message wording, but kept it as a request lock, as it is a lock on the request ID, not on any particular project. (One request can refer to multiple projects and that is why there can be multiple processes launched for the same request and locking is needed)